docs/how-to/platform-console.md

```diff
@@ -0,0 +1,120 @@
+
+### Managing Templates
+
+Managing and updating templates is very similar to updating individual services. You have access to the same settings and options available for services, including:
+
+- Viewing all templates in the Platform Console dashboard.
+- Adding entities to the service template.
+- Adding or removing plugins, including custom plugins.
+- Renaming templates and updating their descriptions.
+- Changing the code generator version for the template.
+
+To edit a template:
+
+1. Select the template from the Platform Console dashboard.
+2. Navigate to the "Settings" tab.
+3. Make your desired changes in the "General" or "Plugins" sections.
+4. All changes are saved automatically.
+
+:::note
+When you update a template, **all services using that template will be automatically updated**, ensuring consistency across your infrastructure.
+:::
+
+### Using Templates
+
+To create a new service based on a template:
+
+1. Click on the "Add Resource" button.
+2. Select "Service from Template" from the available options.
+3. In the modal that appears, choose a specific service template.
+4. Provide a name for your new service.
+5. Click "Create" to generate the new service based on the selected template.
+
+This process ensures that the new service inherits all the pre-configured settings and plugins defined in the template.
+
+![Amplication Create Services From Template](./assets/create-service-from-template.png)
+
+## Private Plugins
+
+Private plugins allow you to create custom functionality specific to your organization's needs. These can be added to templates or directly to services.
+
+:::note
+Creating and managing private plugins requires an Enterprise plan or higher.
+:::
+
+### Creating a Private Plugin
+
+To create a private plugin:
+
+1. In the Platform Console dashboard, navigate to the "Private Plugins" section.
+2. Click on "Create Private Plugin".
+3. Follow the creation wizard to define your plugin's functionality.
+
+## Platform Changes Sidebar
+
+The Platform Console introduces a new **_Platform_ Changes** sidebar that replaces the **_Pending_ Changes** sidebar. It's only visible when you're viewing the Platform Console.
+
+"Platform Changes" shows changes made to templates and private plugins, while "Pending Changes" still shows service-level changes.
+
+:::note
+Platform changes are separate from service-level changes. Make sure to review and commit platform changes separately from service changes.
+:::
+
+![Amplication Create Services From Template](./assets/platform-changes.png)
+
+## Conclusion
+
+The Platform Console feature is in beta, so we encourage you to explore its capabilities and provide feedback to help us improve and refine it.
+
+:::tip
+For any questions or issues related to the Platform Console, please [contact our support team](https://amplication.com/contact) or write your feedback on our [GitHub discussion](https://github.com/amplication/amplication/discussions/8979).
```

Comment:
The support link is broken. Let's avoid the link here and just write "... please contact us or write your..."